docs/security/authorization/spark/build.md

```diff
@@ -31,6 +31,19 @@ After a while, if everything goes well, you will get the plugin finally in two p
 - The main plugin jar, which is under `./extensions/spark/kyuubi-spark-authz/target/kyuubi-spark-authz_${scala.binary.version}-${project.version}.jar`
 - The least transitive dependencies needed, which are under `./extensions/spark/kyuubi-spark-authz/target/scala-${scala.binary.version}/jars`
 
+## Build bundle jar with Apache Maven
+
+Apache Kyuubi also provides the bundle jar for the Spark AuthZ plugin, You can run the AuthZ plugin using just a bundle jar without the additional dependency of jars,
+To build it, `cd` to the root direct of kyuubi project and run:
+
+```shell
+build/mvn clean package -pl :kyuubi-spark-authz-bundle_2.12 -DskipTests -am
+```
+
+After a while, if everything goes well, you will get the plugin finally:
+
+- The bundle authZ plugin jar, which is under `./extensions/spark/kyuubi-spark-authz-bundle/target/kyuubi-spark-authz_${scala.binary.version}-bundle-${project.version}.jar`
```

Comment:
In such case, the name pattern should be 
```
kyuubi-spark-authz-bundle_${scala.binary.version}-${project.version}.jar
```

Also, does the plugin build against one Spark version could be used for all Spark runtimes? If not, should spark binary version be added as part of jar name?

Reply:
> In such case, the name pattern should be

Good catch!



> Also, does the plugin build against one Spark version could be used for all Spark runtimes? If not, should spark binary version be added as part of jar name?

I think so, Could you please double check : ) @bowenliang123